tensilelite/Tensile/Components/MAC_BF16_HPA.py

```diff
@@ -31,79 +31,89 @@ class FMA_BF16_HPA(MAC):
                               "HighPrecisionAccumulate": True}}
 
     def __call__(self, writer, m, innerUnroll):
-        kernel = writer.states.kernel
-        module = Module("FMA_BF16_HPA")
-        module.addComment(self.commentHeader())
-        priority = Component.Priority.find(writer)
+            kernel = writer.states.kernel
```

Comment:
Don't shift line if you didn't have change on it.

Reply:
Apologies that was not intentional. Fixed.